ui/js/search.js

```diff
@@ -0,0 +1,171 @@
+var SEARCH_COURSE_API = 'http://localhost:3000/search-course?searchQuery=';
+var COURSE_MATCHER = new RegExp("^[a-zA-Z]{3}[0-9]{1,3}$");
+var chosenCourses = [];
+
+$(document).ready(function () {
+    var input = '';
```

Comment:
You think it'd be a better idea to make this a local variable within the onclick listener on line 13 since it's not used anywhere else?